mz_crypt_apple.c

```diff
@@ -20,8 +20,12 @@
 
 /***************************************************************************/
 
-/* Avoid use of private API for iOS, Apple does not allow it on App Store. Zip format doesn't need GCM. */
-#if !TARGET_OS_IPHONE
+#ifndef MZ_TARGET_APPSTORE
+#define MZ_TARGET_APPSTORE 0
```

Comment:
I think I should have defaulted it to 1, @nmoinvaz .
Can you add a commit to flip the default value?

Reply:
Done